pkg/sql/create_table.go

```diff
@@ -63,6 +64,13 @@ func (p *planner) CreateTable(ctx context.Context, n *tree.CreateTable) (planNod
 	var sourcePlan planNode
 	var synthRowID bool
 	if n.As() {
+		if containsWindowFunc(p.txCtx, n.AsSource) {
+			return nil, pgerror.NewError(
+				pgerror.CodeFeatureNotSupportedError,
+				"Window functions are not supported in CREATE TABLE AS in 19.1. Consider upgrading to 19.2.",
```

Comment:
1. drop the capital at the beginning of the error message
2. drop the period at the end of the error message
3. use a error hint object to give hints - this can use capitals

For example:

```go
return nil, pgerror.NewError(
     pgerror.CodeFeatureNotSupportError,
     "window functions are not supported in CREATE TABLE AS").SetHintf(
     "This feature was added in CockroachDB 19.1. Consider upgrading.")
```

Reply:
I meant in 19.2 of course